### PR TITLE
Add populate_anticrispr.py

### PR DESCRIPTION
**Pull request by djmitche**
This script populates the restapi_anticrispr table based on the table of
sequences in https://www.nature.com/articles/nmicrobiol201685

This also uses argparse to give nice help output.

### CODE REVIEW

**Comment by djmitche** [comment]
Fixes #122

File: phageAPI/populate_anticrispr.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#! /usr/bin/env python
+
+import os
+from IPython import embed
```
Comment (QuLogic) [inline phageAPI/populate_anticrispr.py]: Not used?

Comment (djmitche) [inline phageAPI/populate_anticrispr.py]: fixed

**Comment by cemyld** [comment]
looks good!

**Pull request merged**